drivers/hmis/lib/form_data/assessments/base_assessment.json

```diff
@@ -1359,234 +1359,303 @@
-              "autofill_when": [
+              "type": "GROUP",
+              "link_id": "disability-table-r6",
+              "text": "Substance Use Disorder",
+              "item": [
                 {
-                  "question": "4.05.2",
-                  "operator": "EXISTS",
-                  "answer_boolean": true
+                  "type": "CHOICE",
+                  "link_id": "4.10.2",
+                  "text": "Substance Use Disorder",
+                  "field_name": "substanceUseDisorder",
+                  "pick_list_reference": "DisabilityResponse"
                 },
                 {
-                  "question": "4.07.2",
-                  "operator": "EXISTS",
-                  "answer_boolean": true
-                },
+                  "type": "CHOICE",
+                  "link_id": "4.10.A",
+                  "text": "Is this a disabling condition?",
+                  "brief_text": "Disabling Condition",
+                  "field_name": "substanceUseDisorderIndefiniteAndImpairs",
+                  "pick_list_reference": "NoYesReasonsForMissingData",
+                  "disabled_display": "PROTECTED",
+                  "enable_behavior": "ANY",
+                  "enable_when": [
+                    {
+                      "question": "4.10.2",
+                      "operator": "EQUAL",
+                      "answer_code": "ALCOHOL_USE_DISORDER"
+                    },
+                    {
+                      "question": "4.10.2",
+                      "operator": "EQUAL",
+                      "answer_code": "DRUG_USE_DISORDER"
+                    },
+                    {
+                      "question": "4.10.2",
+                      "operator": "EQUAL",
+                      "answer_code": "BOTH_ALCOHOL_AND_DRUG_USE_DISORDERS"
+                    }
+                  ]
+                }
+              ]
+            },
+            {
+              "type": "GROUP",
+              "link_id": "disability-table-r7",
+              "text": "Overall Disabling Condition",
+              "_ignore": "HUD defines a Disabling Condition as:<ul><li>A physical, mental, or emotional impairment, including an impairment caused by alcohol or drug abuse, post-traumatic stress disorder, or brain injury that:</li><ul><li>Is expected to be long-continuing or of indefinite duration;</li><li>Substantially impedes the individual's ability to live independently; and</li><li>Could be improved by the provision of more suitable housing conditions.</li></ul></li><li>A developmental disability, as defined in section 102 of the Developmental Disabilities Assistance and Bill of Rights Act of 2000 (42 U.S.C. 15002); or</li><li>The disease of acquired immunodeficiency syndrome (AIDS) or any condition arising from the etiologic agency for acquired immunodeficiency syndrome (HIV).</li></ul>",
```

Comment:
For a future change, but I wonder if we should be storing these as markdown instead of html.  We have a gem called red carpet that can turn markdown into html quite nicely, and using markdown tends to avoid some of the potential errors of using HTML.

Reply:
Once we are using a form editor to generate these JSON files, we can expose a rich text editor for display text fields. At that point I think it would be easier to just store the HTML output directly, since that's what we're rendering (and we won't have the issue of error-prone writing HTML by hand).

Reply:
Sounds perfect